.github/workflows/pull-request.yml

```
@@ -295,7 +295,50 @@ jobs:
+
+          echo "API started successfully"
+
+          # Cleanup
+          docker-compose -f docker-compose.dev.yaml down -v
+          if [ $? -ne 0 ]; then
+          echo "Failed to cleanup containers"
+          exit 1
+          fi
+    env:
+      HEALTH_CHECK_URL: http://localhost:4000
+      COMPOSE_PROJECT_NAME: pr-${{ github.event.pull_request.number }}      
```

yamllint

[error] trailing spaces

(trailing-spaces)